qemu/tests/ovs_quit.py

```diff
@@ -24,33 +22,33 @@ def run(test, params, env):
 
     def get_ovs_ports(ovs):
         cmd = "ovs-vsctl list-ports %s" % ovs
-        return set(utils.system_output(cmd).splitlines())
+        return set(process.system_output(cmd).splitlines())
 
     os_dep.command("ovs-vsctl")
     netdst = params.get("netdst")
-    if netdst not in utils.system_output("ovs-vsctl list-br"):
-        raise error.TestError("%s isn't is openvswith bridge" % netdst)
+    if netdst not in process.system_output("ovs-vsctl list-br"):
+        test.error("%s isn't is openvswith bridge" % netdst)
 
     deps_dir = data_dir.get_deps_dir("ovs")
-    nic_script = utils_misc.get_path(deps_dir, params["nic_script"])
-    nic_downscript = utils_misc.get_path(deps_dir, params["nic_downscript"])
-    params["nic_script"] = nic_script
-    params["nic_downscript"] = nic_downscript
 
-    params["qemu_command_prefix"] = "export SHELL=/usr/bin/bash;"
     params["start_vm"] = "yes"
     params["nettype"] = "bridge"
-    params["nic_model"] = "virtio-net-pci"
+    params["nic_model"] = "virtio-pci"
 
     try:
         ports = get_ovs_ports(netdst)
         env_process.preprocess_vm(test, params, env, params["main_vm"])
         env.get_vm(params["main_vm"])
-    except virt_vm.VMStartError:
-        ports = get_ovs_ports(netdst) - ports
-        if ports:
-            for p in ports:
-                utils.system("ovs-vsctl del-if %s %s" % (netdst, p))
-            raise error.TestFail("%s not delete after qemu quit." % ports)
+    except (virt_vm.VMCreateError, virt_vm.VMStartError) as err_msg:
+        match_error = "Parameter 'driver' expects"
+        output = getattr(err_msg, 'reason', getattr(err_msg, 'output', ''))
+        if match_error in output:
+            ports = get_ovs_ports(netdst) - ports
+            if ports:
+                for p in ports:
+                    process.system_output("ovs-vsctl del-if %s %s" % (netdst, p))
```

Comment:
In my opinion, put the cmd in cfg file means the cmd is configurable(like RHEL and Win guest use the different cmd), or at least there's possibility that this cmd will be modified in the future somehow. But in this case, the cmd will not be modified. So I think it's not neccesary to move the cmd to cfg file.
In addition, in such cases, whether to put the cmd in code or in cfg file is up to the coder. It's not like the configurable cmd, we should put into cfg file.